### ISSUE TITLE
[Bug] Cannot add an AAD user to Project security group if user not part of the organization

### ISSUE THREAD

**Issue by carmando62** [Azure/azure-devops-cli-extension#918]
Unable to add add AAD users to a new project's security groups where users are not part of the organization yet.  The CLI documentation shows --member-id that the "Descriptor of the group or Email Id of the user to be added."  First off it should be member Id as Descriptor of the group goes under --group-id. Using the --member-id but this is only valid if the user is already a member of the organization.  Email Id is mentioned which makes more sense however, even though it is in the documentation the CLI does not recognize this option.

**To Reproduce**
Azure Cli Version: 2.0.77

Azure-Devops extension version: 0.13.0

1. login using az login
2. setup defaults..
3. Run command...

This does not work because you cannot provide a member Id for a user that is not part of the organization yet because you cannot query the user and get and Id.
az devops security group membership add --group-id $teamDescriptor --member-id myName --org $org

I have tried also tried this to see if the --email-id would work.
az devops security group membership add --group-id $teamDescriptor --email-id myName@testhome.com --org $org
.

**Comment by ishitam8** [comment]
@carmando62 ,

This behavior is by design. We can't resolve identity to its correct azure devops identifier by just name or email-ID. Hence, its mandatory to add the user first to the organization and then add to groups.
Please add the AAD user to org first and then run the command with email-ID and it should work fine.

>  First off it should be member Id as Descriptor of the group goes under --group-id. 

You can add one group as a member of another group too. That's why it says "Descriptor of the group or Email Id of the user to be added."

>Using the --member-id but this is only valid if the user is already a member of the organization. Email Id is mentioned which makes more sense however, even though it is in the documentation the CLI does not recognize this option.

Will it help if help text says something like 
`Descriptor of the group or Email Id of the user to be added. User should already be a part of the organization.`


      PS C:\> az devops security group membership add -h
      This command is from the following extension: azure-devops

        Command
        az devops security group membership add : Add membership.

        Arguments
            --group-id  [Required] : Descriptor of the group to which member needs to be added.
            --member-id [Required] : Descriptor of the group or Email Id of the user to be added.
                                  User should already be a part of the organization.

**Comment by carmando62** [comment]
Thank you.  The problem then for me was adding the new users to
organization. So I went to see what license types all the other users were
and attempted add a new user to the Org using the same license toy as
everyone else (I was initially trying "professional")  I used express and
it worked.  Thank you for getting me headed in the right direction. I am
not blocked anymore.  Also thanks for the clarification.  I do thing the
documentation could be a little in depth.. Greatly appreciate your response,

Mike
